### PR TITLE
makes ocular warden code less shit by 2 characters and makes them not attack stunned people for real

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

did this like 2 years ago but as it turns out the Stun effect is rarely used in favor of Paralyze so that's now checked as well
also

### Why is this change good for the game?
less shitcode good more function as intended also good
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
ocular wardens won't attack people who are stunned any more
# Changelog

:cl:  
tweak: ocular wardens won't attack people who are stunned
/:cl:

### CODE REVIEW

**Review by redmoogle — APPROVED** [review]
Aight, cool. no reason this wont work

**Comment by ChesterTheCheesy** [comment]
New meta is cuffing yourself then detonating bomb from middle of ark?

**Comment by Hopekz** [comment]
You should maybe add an exception to the toy cuffs if you think it'll be an issue

**Comment by redmoogle** [comment]
> 
> 
> You should maybe add an exception to the toy cuffs if you think it'll be an issue

> You get past the 5 billion wardens
> A cultist sees you
> A cultist sees you _(yes this is intentional)_
> you get stabbed to death while still cuffed

**Comment by Hopekz** [comment]
> > 
> > 
> > You should maybe add an exception to the toy cuffs if you think it'll be an issue
> 
> > You get past the 5 billion wardens
> > A cultist sees you
> > A cultist sees you _(yes this is intentional)_
> > you get stabbed to death while still cuffed

Except you forgot the part where you can take off your own toy cuffs in 1 second

**Comment by SomeguyManperson** [comment]
take off toy cuffs then get hit by 4 wardens at once and die

**Comment by Hopekz** [comment]
> take off toy cuffs then get hit by 4 wardens at once and die

In his example they already got past the 5 billion wardens
